singularity-eos/eos/eos_eospac.hpp

```diff
@@ -1251,15 +1251,14 @@ inline std::size_t EOSPAC::SetDynamicMemory(char *src, const SharedMemSettings &
   static_assert(sizeof(char) == sizeof(EOS_CHAR), "EOS_CHAR is one byte");
   EOS_INTEGER NTABLES[] = {NT};
   EOS_INTEGER error_code = EOS_OK;
-  eosCheckError(error_code, "eos_DestroyAll", Verbosity::Debug);
 #ifdef SINGULARITY_EOSPAC_ENABLE_SHARED_MEMORY
   PORTABLE_ALWAYS_REQUIRE(
       stngs.data != nullptr,
       "EOSPAC with shared memory active requires a shared memory pointer");
-  // JMM: EOS_BOOLEAN is an enum with EOS_FALSE=0 and EOS_TRUE=1.
+  // EOS_BOOLEAN is not a type alias
+  EOS_BOOLEAN root = stngs.is_domain_root ? EOS_TRUE : EOS_FALSE;
```

Comment:
The need for this change is a bit surprising to me, but I trust you've found that the old method failed

Reply:
I might be missing something---for example a static cast might work. It appears to be an enum under the hood, so probably `EOS_TRUE` evaluates to 1 and `EOS_FALSE` to 0 but C++ booleans and C booleans aren't the same so I just decided to play it safe.